src/style.css

```diff
@@ -1,3 +1,83 @@
 body {
   margin: 0;
+  --main-color: #00acdc;
+  font-weight: 500;
+}
+
+li.a {
```

Comment:
1'st point in checklist, also use classes

Reply:
you already have classname for a tag, but init font better in html selector for all tags